docroot/profiles/custom/cgov_site/modules/custom/cgov_saml_auth_config/cgov_saml_auth_config.install

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @file
+ * Install file for cgov_saml_auth_config.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function cgov_saml_auth_config_install() {
```

Comment:
This smells like it should just be an actual config override. https://www.drupal.org/docs/8/api/configuration-api/configuration-override-system

If it is really just setting values, then please do not use the word override.

Reply:
As discussed, this is just a one-time setting of the configuration.  Comments and code have been modified to clarify.